woocommerce-services.php

```diff
@@ -530,11 +530,29 @@ public function init_core_wizard_shipping_config() {
 
 		public function init_core_wizard_payments_config() {
 			$stripe_settings = get_option( 'woocommerce_stripe_settings', false );
-			$stripe_enabled  = is_array( $stripe_settings )
+
+			$should_create_stripe_account  = is_array( $stripe_settings )
```

Comment:
When reading this, `$should_create_stripe_account` sounds like it should indicate whether a Stripe account should be created under all conditions. However, we don't want to create a Stripe account based on `$stripe_already_connected`.

I think renaming this variable from `$should_create_stripe_account` to `$user_elected_to_create_stripe_account` would make this distinction more obvious without losing readability later in line 555:

`if ( $user_elected_to_create_stripe_account && is_plugin_active( 'woocommerce-gateway-stripe/woocommerce-gateway-stripe.php' ) ) {`